src/video/wiiu/SDL_wiiuvideo.c

```diff
@@ -59,11 +59,17 @@
 #define DRC_SCREEN_WIDTH    854
 #define DRC_SCREEN_HEIGHT   480
 
+static SDL_bool running = SDL_FALSE;
+
 static int WIIU_ForegroundAcquired(_THIS)
 {
 	WIIU_VideoData *videodata = (WIIU_VideoData *) _this->driverdata;
```

Comment:
You shouldn't access `_this` here if `running` is false. Can you change it to:
```c
	WIIU_VideoData *videodata;
	SDL_Window* window;

	if (!running) {
		return 0;
	}

	videodata =  (WIIU_VideoData *) _this->driverdata;
	window = _this->windows;
```

Reply:
Done.